deb/build

```diff
@@ -40,4 +40,22 @@ fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
 	--depends libtool \
 	--deb-changelog "$changelog" \
 	--deb-no-default-config-files \
-	install/=/
+	install/usr/lib/=/usr/lib/
+
+pkgname=libdrizzle-redux-dev
```

Comment:
Not only headers, the regular package should provide only the library with version included: libdrizzle-redux.so.x.y.z (the real file), libdrizzle-redux.so.x.y and libdrizzle-redux.so.x (symlink to the real lib), while the -dev package should provide a symlink without the version (libdrizzle-redux.so). The -dev package should ideally include the static lib too (libdrizzle-redux.a).

See for example:
* http://packages.ubuntu.com/trusty/amd64/libncurses5-dev/filelist / http://packages.ubuntu.com/trusty/amd64/libncurses5/filelist
* http://packages.ubuntu.com/wily/amd64/liblzo2-dev/filelist / http://packages.ubuntu.com/wily/amd64/liblzo2-2/filelist

Also, providing a -dbg package would be quite nice.